scvi/module/_mrdeconv.py

```diff
@@ -43,8 +43,14 @@ class MRDeconv(BaseModuleClass):
         Mean parameter for each component in the empirical prior over the latent space
     var_vprior
         Diagonal variance parameter for each component in the empirical prior over the latent space
+    weight_vprior
+        Weight parameter for each component in the empirical prior over the latent space
```

Comment:
call this mp_vprior?

mixture proportion of each component in the empirical prior over the latent space